### PR TITLE
Added Batch proposals support

### PR DESCRIPTION
**Pull request by ernestognw**
## Description

Adjusted types according to backend

### CODE REVIEW

**Comment by pasevin** [comment]
This is still in progress, right?

File: packages/admin/src/models/proposal.ts [hunk 1]
```diff
@@ -41,3 +59,18 @@ export interface ProposalFunctionInputType {
...
+  network: Network;
+  decimals: number;
+  type: 'ERC20';
+}
+export interface NativeCurrency {
+  name: string;
+  symbol: string;
+  decimals: number;
+  type: 'native';
+}
```
Comment (shahnami) [inline packages/admin/src/models/proposal.ts:76]: Where did these come from?

**Review by shahnami — APPROVED** [review]
LGTM

File: packages/admin/src/models/proposal.ts (hunk 1 above)
Comment (ernestognw) [reply packages/admin/src/models/proposal.ts:76]: Also the backend, it should be in the response, we just were not typing it on the client

**Pull request merged**